wrappers/core18.go

```diff
@@ -652,12 +651,8 @@ func writeSnapdDbusActivationOnCore(s *snap.Info) error {
 
 	content := make(map[string]osutil.FileState, len(dbusSessionServices)+1)
 	for _, service := range dbusSessionServices {
-		filePathInSnap := filepath.Join(s.MountDir(), "usr/share/dbus-1/services", service)
-		if !osutil.FileExists(filePathInSnap) {
-			continue
-		}
 		content[service] = &osutil.FileReference{
-			Path: filePathInSnap,
+			Path: filepath.Join(s.MountDir(), "usr/share/dbus-1/services", service),
```

Comment:
This is the only other non-trivial change. Seems like the previous change was made to skip services that don't exist. This makes things more robust against errors, but maybe lets silent failures slip by unnoticed.

Reply:
This change is less obvious to exactly know the meaning off. I *really* need the test suite to run to have just an ounce of confidence in this

Reply:
I think the change is correct in the sense that it support removing files which is what we want, but I'm confused because I see the other pattern elsewhere in this file which I thought would not be the case, I need to do some blame on master to understand the state of things

Reply:
sorry, I misread the code, we do not want this change, the code in master is what we want